src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -54,6 +54,18 @@ public void updateItem(TurboIssue issue, boolean empty) {
         this.setId(issue.getRepoId() + "_col" + parentPanelIndex + "_" + issue.getId());
     }
 
+    private void updateStyleToMatchStatus(TurboIssue issue){
+        if (!issue.isOpen()){
+            if (!isCurrentStyleClosed) {
+                getStyleClass().add(closedStyle);
+            }
+        } else {
+            if (isCurrentStyleClosed) {
+                getStyleClass().remove(closedStyle);
+            }
+        }
+    }
```

Comment:
I'm not sure if those variables should be at instance level. Can't we keep them inside the method?